specification/ai/Azure.AI.Projects/agents/tools/models.tsp

```diff
@@ -130,6 +140,19 @@ model FunctionDefinition {
   parameters: unknown;
 }
 
+@doc("The definition of Azure function.")
+model AzureFunctionDefinition {
+  @doc("The definition of azure function and its parameters.")
+  function: FunctionDefinition;
+
+  @doc("Input storage queue. The queue storage trigger runs a function as messages are added to it.")
+  @encodedName("application/json", "input_binding")
+  inputBinding: AzureStorageQueueBinding;
```

Comment:
I believe this should be:

```suggestion
  inputBinding: AzureFunctionBinding;
```

(because AzureFunctionBinding seem to be an extensibility point/AzureStorageQueueBinding happens to be *one* (and currently the only) binding type,  but it is itself a polymorphic type.

Reply:
Fixed.